Rakefile

```diff
@@ -93,11 +93,23 @@ task :student_pre do
   %x{printf '\nsupersede domain-search "puppetlabs.vm";\n' >> /etc/dhcp/dhclient-eth0.conf}
 end
 
+desc "Puppetfactory VM pre-install setup"
+task :puppetfactory_pre do
+  # Set the dns info and hostname; must be done before puppet
+  cputs "Setting hostname puppetfactory.puppetlabs.vm"
+  %x{hostname puppetfactory.puppetlabs.vm}
+  cputs  "Editing /etc/hosts"
+  %x{sed -i "s/127\.0\.0\.1.*/127.0.0.1 puppetfactory.puppetlabs.vm puppetfactory localhost localhost.localdomain localhost4/" /etc/hosts}
+  cputs "Editing /etc/sysconfig/network"
+  %x{sed -ie "s/HOSTNAME.*/HOSTNAME=puppetfactory.puppetlabs.vm/" /etc/sysconfig/network}
+  %x{printf '\nsupersede domain-search "puppetlabs.vm";\n' >> /etc/dhcp/dhclient-eth0.conf}
+end
+
 desc "Apply bootstrap manifest"
 task :build do
  system('gem install r10k --no-RI --no-RDOC')
  Dir.chdir('/usr/src/puppetlabs-training-bootstrap') do
-  system('RUBYLIB="/usr/src/puppet/lib:/usr/src/facter/lib:/usr/src/hiera/lib" PATH=$PATH:/usr/src/puppet/bin r10k puppetfile install')
+  system('RUBYLIB="$RUBYLIB:/usr/src/puppet/lib:/usr/src/facter/lib:/usr/src/hiera/lib" PATH="$PATH:/usr/local/bin:/usr/src/puppet/bin" r10k puppetfile install')
```

Comment:
It seems like we may want `$RUBYLIB` at the end of the list, instead of the beginning. That would ensure that we had precedence if there was any duplication. Maybe not a concern?